includes/Modules/AdSense/Settings.php

```diff
@@ -88,6 +90,8 @@ public function register() {
 			)
 		);
 
+		$this->register_owned_keys( $this->get_owned_keys() );
```

Comment:
See my last comment, if we changed that, here (and below) we would only need to call `$this->register_owned_keys()`.

Reply:
Updated.